Updated MQTT testing documentation (#14537)
home-assistant/home-assistant.github.io: source/_docs/mqtt/testing.markdown

mosquitto_pub -h 127.0.0.1 -t home-assistant/switch/1/on -m "Switch is ON"
```

> Another way to send MQTT messages by hand is to use "MQTT" Integration in the Frontend. Choose the "Configuration" tab and click "Integrations" and click the "Configure" option under the MQTT integration. Enter something similar to the example below into the "Topic" field.

Reviewer: ```suggestion
Another way to send MQTT messages manually is to use the "MQTT" Integration in the frontend. Choose the "Configuration" tab, click "Integrations" and click the "Configure" option under the "MQTT" integration. Enter something similar to the example below into the "topic" field under "Publish a packet*.
```

Author: Thanks for the additional cleanup.
